docker/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# base Dockerfile: PyTorch nightly docker
+# ghcr.io/pytorch:pytorch-nightly
+FROM ghcr.io/pytorch/pytorch-nightly:latest
```

Comment:
We can use build-args like
```dockerfile
ARG FROM_IMAGE=ghcr.io/pytorch/pytorch-nightly:latest
FROM ${FROM_IMAGE}
```
so it's easy to change base image from `docker build` command line.